CMakeUserPresets.json

```diff
@@ -0,0 +1,3 @@
+{

```

Comment:
This empty file would not be necessary if generating the file could be prevented by a global.conf. 
see also: https://github.com/conan-io/cmake-conan/issues/484#issuecomment-1628878720